src/main/java/net/java/otr4j/session/Session.java

```diff
@@ -648,12 +648,29 @@ private String handleDataMessage(DataMessage data) throws OtrException {
                 if (tlvs != null && tlvs.size() > 0) {
                     for (TLV tlv : tlvs) {
                         switch (tlv.getType()) {
-                            case TLV.DISCONNECTED:
+                            // TVL.PADDING // TLV0
```

Comment:
makes sense, fixed.  I also added a `default:` to throw a warning message to aid debugging.